packages/server-common/src/utils/sentry-wrapper.ts

```diff
@@ -0,0 +1,23 @@
+import * as Sentry from '@sentry/serverless';
+import { Handler } from 'aws-lambda';
+
+export interface SentryConfig {
+  currentRevision: string | undefined;
+  environment: string | undefined;
+  sentryDsn: string | undefined;
+  sentryTraceSampleRate: number | undefined;
+}
+
+export const sentryWrapperImpl = (
```

Comment:
i would suggest that you make this a factory:
```suggestion
export const sentryWrapperFactory = (config: SentryConfig) => (handler: Handler) => .....
```

and then in the crn/gp2 utils just do
```typescript
export const sentryWrapper = sentryWrapperFactory({
    currentRevision,
    environment,
    sentryDsn,
    sentryTraceSampleRate,
  });
```